robosuite/renderers/mjviewer/mjviewer_renderer.py

```diff
@@ -1,15 +1,28 @@
+import cv2
+import numpy as np
 from mujoco import viewer
 
 
 class MjviewerRenderer:
-    def __init__(self, env, camera_id=None, cam_config=None):
+    def __init__(self, env, camera_id=None, cam_config=None, sim=None):
         self.env = env
         self.camera_id = camera_id
         self.viewer = None
         self.camera_config = cam_config
 
+        self.width = 1280
+        self.height = 800
```

Comment:
Will these values be flexible to change? We want some configuration more flexible than writing fixed values like this.

Reply:
I just follow the structure of OpenCVRenderer (https://github.com/ARISE-Initiative/robosuite/blob/6ec07a1ede31e1ede93c3d29fce4a7fefe0cad1f/robosuite/utils/opencv_renderer.py#L11). Maybe you could pass the configuration through some args like screen_config?

Reply:
controlling the width / height of the mjviewer is currently not supported, so this feature is not yet applicable here